quickwit-directories/src/bundle_directory.rs

```diff
@@ -264,4 +253,34 @@ mod tests {
 
         Ok(())
     }
+
+    #[tokio::test]
+    async fn test_stream_split_to_bundle_and_open() -> anyhow::Result<()> {
+        let temp_dir = tempfile::tempdir()?;
+        let test_filepath1 = temp_dir.path().join("f1");
+        let test_filepath2 = temp_dir.path().join("f2");
+
+        let mut file1 = File::create(&test_filepath1)?;
+        file1.write_all(&[123, 76])?;
+
+        let mut file2 = File::create(&test_filepath2)?;
+        file2.write_all(&[99, 55, 44])?;
+
+        let split_streamer = get_split_streamer(
+            &[test_filepath1.clone(), test_filepath2.clone()],
+            &[1, 2, 3],
```

Comment:
Not even a nitpick. just a matter of taste:
 
For meaninless `&[u8]` in unit test, I like to use bytestrings : b"234234".

Merits
- compactness
- it tells the reader that this is a &[u8] (right now the array could be &[usize])

Reply:
It can be nice, my only issue with it is that the exact bytes behind the string may not be clear